config.md

```diff
@@ -352,6 +352,52 @@ For Linux-based systems, the `process` object supports the following process-spe
       for `initial`. If omitted or empty, runtime SHOULD NOT change process'
       CPU affinity after the process is moved to container's cgroup, and the
       final affinity is determined by the Linux kernel.
+* **`landlock`** (object, OPTIONAL) specifies the Landlock unprivileged access control settings for the container process.
```

Comment:
I think this doesn't belong to this file. This should be in config-linux. The same with the secion of the struct this is, this should be inside the Linux section, as seccomp is. Not as a general feature, as this is a linux-only feature.

Reply:
Ouch, there is a section for Linux here too. Sorry!